### ISSUE TITLE
Calculating f1 score 

### ISSUE THREAD

**Issue by AbdulRafay** [lonePatient/Bert-Multi-Label-Text-Classification#55]
@lonePatient Can you please guide me in calculating f1 score and confusion matrix with your code? where can i get the y_pred and y_pred as described [here](https://scikit-learn.org/stable/modules/generated/sklearn.metrics.multilabel_confusion_matrix.html).

Best regards

**Comment by AbdulRafay** [comment]
I passed it in the trainer 

```
trainer = Trainer(n_gpu=args.n_gpu,
                      model=model,
                      epochs=args.epochs,
                      logger=logger,
                      criterion=BCEWithLogLoss(),
                      optimizer=optimizer,
                      lr_scheduler=lr_scheduler,
                      early_stopping=None,
                      training_monitor=train_monitor,
                      fp16=args.fp16,
                      resume_path=args.resume_path,
                      grad_clip=args.grad_clip,
                      model_checkpoint=model_checkpoint,
                      gradient_accumulation_steps=args.gradient_accumulation_steps,
                      batch_metrics=[AccuracyThresh(thresh=0.5)],
                      epoch_metrics=[AUC(average='micro', task_type='binary'),
                                     MultiLabelReport(id2label=id2label), F1Score(task_type='binary', average='macro')])
```
